app/config/Module.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import com.google.inject.AbstractModule
+
+class Module extends AbstractModule {
+
+  override def configure(): Unit = {
+
+    bind(classOf[AppConfig]).asEagerSingleton()
```

Comment:
Can you make this line match what's specified in the other backends?
`bind(classOf[AppConfig]).to(classOf[AppConfig]).asEagerSingleton()`
https://github.com/hmrc/income-tax-cis/blob/main/app/config/Modules.scala#L24
https://github.com/hmrc/income-tax-expenses/blob/main/app/config/Modules.scala#L24
https://github.com/hmrc/income-tax-employment/blob/main/app/config/Modules.scala#L24

Could you also rename the file to `Modules`, just to keep things consistent with the naming used in the other SA submission repos.

Reply:
There is also no need to put both, AppConfig is a class and it will be bind to itself.